spring-jdbc/src/main/java/org/springframework/jdbc/support/SQLErrorCodeSQLExceptionTranslator.java

```diff
@@ -360,38 +360,44 @@ protected DataAccessException createCustomException(
 			// invoke constructor
 			Constructor<?> exceptionConstructor;
 			switch (constructorType) {
```

Comment:
Guess that should be replaced with single `return switch` with `yield`s inside.

Reply:
done